easybuild/tools/github.py

```diff
@@ -376,14 +392,22 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
     for (dirpath, _, filenames) in os.walk(path):
         tmp_files.extend([os.path.join(os.path.basename(dirpath), f) for f in filenames])
 
-    if not sorted(tmp_files) == sorted(all_files):
-        raise EasyBuildError("Not all patched files were downloaded to %s: %s vs %s", path, tmp_files, all_files)
+    for patched in all_files:
+        if not patched in tmp_files:
+            raise EasyBuildError("Couldn't find file in %s: %s", path, patched)
 
-    ec_files = [os.path.join(path, f) for f in tmp_files]
+    ec_files = []
+    for patched in patched_files:
+        if os.path.exists(os.path.join(path, patched)):
+            ec_files.append(os.path.join(path, patched))
+        else:
+            raise EasyBuildError("Coudln't find path to patched file %s", os.path.join(path, f))
```

Comment:
isn't this the same check as above? checking once is enough (keep this one, drop the separate loop above?)